index.js

```diff
@@ -43,6 +43,14 @@ class Conf {
 		this.encryptionKey = options.encryptionKey;
 		this.path = path.resolve(options.cwd, `${options.configName}.${options.fileExtension}`);
 
+		if (options.validate) {
+			try {
+				JSON.parse(fs.readFileSync(this.path));
+			} catch (ex) {
+				throw ex;
+			}
```

Comment:
I don't understand why you're wrapping this in try/catch? It has no purpose.

And I think this check would be better done here instead: https://github.com/sindresorhus/conf/blob/8ce463d1ecda92b53595904e01868012c5d11659/index.js#L152

Reply:
I added the try/catch in order to throw the exception to the consumer of the library so that he can handle it appropriately.

I added the check in the constructor because `options` object is available there and I thought that it would be a good choice to validate early on.